webdriver/tests/support/fixtures.py

```diff
@@ -174,8 +174,12 @@ async def session(capabilities, configuration):
 
     # Enforce a fixed default window size and position
     if _current_session.capabilities.get("setWindowRect"):
-        _current_session.window.size = defaults.WINDOW_SIZE
-        _current_session.window.position = defaults.WINDOW_POSITION
+        # Only resize and repositioning if needed to workaround a bug for Chrome:
+        # https://bugs.chromium.org/p/chromedriver/issues/detail?id=4642#c4
+        if _current_session.window.size != defaults.WINDOW_SIZE:
+            _current_session.window.size = defaults.WINDOW_SIZE
+        if _current_session.window.position != defaults.WINDOW_POSITION:
+            _current_session.window.position = defaults.WINDOW_POSITION
```

Comment:
question: do we need to add the same to the bidi_session fixture?

Reply:
Oh, good find! I completely missed that. I'll push an update after checking why some tests failed.